src/promptflow/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 ### Bugs Fixed
 - [Flow build] Fix flow build file name and environment variable name when connection name contains space.
 
+### Other Changes
+- [Executor][Internal] Handle JSONDecodeError exception when executor is resolving the inputs type convert
```

Comment:
This part conflicts with the other PR. Do we need to add separate items for each PR in the changelog?
I think we can have a general one like "Refine exception messages".